### PR TITLE
Update with Bitbucket context

### PR DESCRIPTION
**Pull request by borsos-ilo**

### CODE REVIEW

**Comment by JEverhart383** [comment]
BitBucket planning April 13th, maybe live April 20th. Unsure if we'll launch both features or just one integration.

File: atlas/additional-guides/troubleshooting-builds.mdx [hunk 1]
```diff
@@ -23,27 +23,29 @@ Your build will be displayed in the `Builds` table with the status `Cancelled` -
...
 
 As the build progresses, it can either succeed or fail at any stage in the build -> deploy checklist. If the project fails to build, the build details on your Atlas dashboard will look like this:
 
 ![Headless App Failed in Building](./assets/img/troubleshooting-builds/code-deploy-failed.png)
 
 Since Atlas accepts a wide variety of frontend application frameworks (Faust.js, Next.js, Nuxt, Svelte, React, Remix, etc.), there are many different ways that building a particular application could fail. Since there are few commonalities in these failure modes, this guide will focus on giving you the tools to inspect and diagnose your own app's failure state.
 
-After you've pushed a failing build, there are quick ways to start troubleshooting your app from the Atlas dashboard by inspecting the GitHub commit or looking at the build logs directly.
+After you've pushed a failing build, there are quick ways to start troubleshooting your app from the Atlas dashboard by inspecting the commit or looking at the build logs directly.
```
Comment (richard-wpengine) [inline atlas/additional-guides/troubleshooting-builds.mdx:34]: repository commit

**Review by richard-wpengine — DISMISSED** [review]
LGTM, besides comment on changing `commit` to `repository commit`, for consistency.

**Comment by JEverhart383** [comment]
@ikedracka and @richard-wpengine I created an environment to review some of these changes since they are pretty large in scope: https://hjge992vplk09vysbgfj2arjw.js.wpenginepowered.com/docs/atlas/overview

**Pull request merged**